### ISSUE TITLE
[1.5.2.1][osx] keeps asking 'to accept incoming network connections'.

### ISSUE THREAD

**Issue by rjsmith** [meteor/meteor#9139]
I have just upgraded my Meteor application from 1.5.2 to 1.5.2.1 and immediately noticed that OSX 10.11.6 is prompting "Do you want the application “node” to accept incoming network connections? Allow/Deny" every time Meteor re-starts the server application after a re-build.

I normally only see this prompt once after upgrading a Meteor version. I have not seen this behaviour on 1.5.2 or earlier versions. 

Steps to reproduce:

1. Edit a client - only file. Save. Meteor does a client-only rebuild, no "accept incoming" prompt.
2. Edit a server - only file (I tried the in the /server folder). Save. Meteor does a server re-build and re-starts the server and OSX shows the "accept incoming" prompt.
3. Edit a client or server-side file in a local package. Save. Meteor re-builds and re-starts the server and OSX shows the "accept incoming" prompt.

The applications /.meteor/release file has "METEOR@1.5.2.1"

**Comment by abernix** [comment]
I'm currently working on this and should have a solution soon.

## Easy Fix for macOS Users

Copy the `node` binary from the 1.5.2 release into your 1.5.2.1 release using the following (long) command:

    cp ~/.meteor/packages/meteor-tool/1.5.2/mt-os.osx.x86_64/dev_bundle/bin/node ~/.meteor/packages/meteor-tool/1.5.2_1/mt-os.osx.x86_64/dev_bundle/bin/node

It's built from the same source and has no other changes for macOS users (there were changes for Linux users; see #9068).  Meteor 1.5.2 will need to be installed for this command to work, so if you receive an error when running it, just run `meteor create --release 1.5.2 test-app-safe-to-delete` somewhere to put the binary in place first.

## Prognosis

I know why the error is happening every time, and it's related to the code-signing signature on the `node` binary included in 1.5.2.1.  I've confirmed that the binary is identical to that which was built on our build Mac, but the build Mac hardware _itself_ changed so something else happened in the compilation process.  This is could be the root cause of #9136 as well.

#### 1.5.2.1 binary
```sh
$ codesign --verify -vv ~/.meteor/packages/meteor-tool/1.5.2_1/mt-os.osx.x86_64/dev_bundle/bin/node
/Users/jesse/.meteor/packages/meteor-tool/1.5.2_1/mt-os.osx.x86_64/dev_bundle/bin/node: invalid signature (code or signature have been modified)
In architecture: x86_64
```

#### 1.5.2 binary
```sh
$ codesign --verify -vv ~/.meteor/packages/meteor-tool/1.5.2/mt-os.osx.x86_64/dev_bundle/bin/node
```

More details and a fix with a test to prevent this in the future coming soon.

**Comment by rjsmith** [comment]
Hi @abernix .  Thanks. I can confirm the above workround has resolved the problem on my dev machine.

**Comment by abernix** [comment]
Just FYI, I ran into delays initially because SourceForge downloads were down and they host the [ICU](//icu-project.org) (International Components for Unicode) dependencies necessary to build `node`, but later on due to some bizarre `codesign` issues.  The full fix is still being investigated, but we hope to have an answer soon.

**Comment by m-salamon** [comment]
having the same issue when I updated to `node v8.6.0`

**Comment by abernix** [comment]
A release candidate for Meteor 1.5.2.2 has been released which includes this change.  Please help test it and confirm whether this issue is fixed!

    meteor update --release 1.5.2.2-rc.0

**Comment by hwillson** [comment]
Just chiming in @abernix - I've confirmed the 1.5.2.2 RC addresses this issue for me. Thanks!

**Comment by rjsmith** [comment]
Hi @abernix . Yes, after upgrading to 1.5.2.2-rc.0, I'm not seeing the OSX prompt when the meteor server restarts after a rebuild.  Thanks!

**Comment by benjamn** [comment]
This should be fixed if you update to Meteor 1.5.2.2 (`meteor update --release 1.5.2.2`). Thanks everyone!
